tests/relay_integration/lang/javascript/test_plugin.py

```diff
@@ -3027,3 +3075,75 @@ def test_expansion_with_release_and_malformed_sourcemap(self, process_with_symbo
             "type": "js_invalid_source",
             "url": "http://example.com/file.malformed.sourcemap.js",
         }
+
+    @requires_symbolicator
+    @pytest.mark.symbolicator
+    @with_feature("organizations:sourcemaps-bundle-flat-file-indexing")
+    @override_options({"symbolicator.sourcemaps-bundle-index-sample-rate": 1.0})
+    def test_bundle_index(self, process_with_symbolicator):
+        if not process_with_symbolicator:
+            return
+
+        project = self.project
+        release = Release.objects.create(organization_id=project.organization_id, version="abc")
+        release.add_project(project)
+
+        file_a = make_compressed_zip_file(
+            {
+                "files/_/_/file.min.js": {
+                    "url": "~/file.min.js",
+                    "type": "minified_source",
+                    "content": load_fixture("file.min.js"),
+                    "headers": {
+                        "sourcemap": "file.sourcemap.js",
+                    },
+                },
+                "files/_/_/file.sourcemap.js": {
+                    "url": "~/file.sourcemap.js",
+                    "type": "source_map",
+                    "content": load_fixture("file.wc.sourcemap.js"),
+                },
+            },
+        )
+
+        upload_bundle(file_a, self.project, release.version)
+
+        data = {
+            "timestamp": self.min_ago,
+            "message": "hello",
+            "platform": "javascript",
+            "release": "abc",
+            "exception": {
+                "values": [
+                    {
+                        "type": "Error",
+                        "stacktrace": {
+                            "frames": [
+                                {
+                                    "abs_path": "http://example.com/file.min.js",
+                                    "filename": "file.min.js",
+                                    "lineno": 1,
+                                    "colno": 39,
+                                },
+                            ]
+                        },
+                    }
+                ]
+            },
+        }
+
+        event = self.post_and_retrieve_event(data)
+
+        assert "errors" not in event.data
+
+        exception = event.interfaces["exception"]
+        frame_list = exception.values[0].stacktrace.frames
+
+        frame = frame_list[0]
+        assert frame.function == "add"
+        assert frame.filename == "file1.js"
+        assert frame.lineno == 3
+        assert frame.colno == 9
+
+        assert frame.data["resolved_with"] == "index"
```

Comment:
Are we using the resolved with `index` for both db and flat file index?

Reply:
yes, although one of them is passed down by the lookup API, and the other one is coming from symbolicator internally. Maybe it might be worth defining two separate things for it, but for now it does not matter that much.